docs/source/reference/respawn/player.rst

```diff
@@ -11,10 +15,18 @@ Functions for getting player, and methods of the player object
 
 .. cpp:function:: array<entity> GetPlayerArray()
 
+.. cpp:function:: array<entity> GetPlayerArrayEx( string affected, int team, vector origin, float radius )
+
+    returns a list of every player in radius relative to origin. I only found ``"any"`` to work and assume the first parameter describes the affected types.
```

Comment:
I would recommend using more general language here. I'm not exactly sure what the `"any"` here is supposed to refer to. Maybe something like:
"It appears only the string `any` works".
Or maybe just remove it alltogether